src/plugins/data_source/server/plugin.ts

```diff
@@ -107,4 +150,28 @@ export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourc
       };
     };
   };
+
+  private async logAuditMessage(
+    auditorPromise: Promise<Auditor>,
+    dataSourceId: string,
+    request: OpenSearchDashboardsRequest
+  ) {
+    const auditor = await auditorPromise;
+    const auditMessage = this.getAuditMessage(request, dataSourceId);
+
+    auditor.add({
+      message: auditMessage,
+      type: 'opensearch.dataSourceClient.call.internalUser',
```

Comment:
this doesn't make sense, it's always current user, not internal user. Type here should by dynamic, possible values for example `creation, access, change, deletion`.  reference: https://www.elastic.co/guide/en/kibana/7.11/xpack-security-audit-logging.html#field-event-type

Or come up with some better candidates

Reply:
good catch, but still going to update the string and limit the scope for this pr to be some static string for now.